.circleci/config.yml

```diff
@@ -1421,3 +1430,4 @@ workflows:
       - installation-tests-carthage
       - installation-tests-xcode-direct-integration
       - installation-tests-receipt-parser
+      - deploy-purchase-tester-dry-run
```

Comment:
dry run job in all-tests so that we know if we're breaking purchase tester (I didn't realize that I had broken it until Toni mentioned it in a PR comment)